app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -188,18 +192,19 @@ def update_form_with_classification_code(classification_code)
   end
 
   def log_max_cfi_metrics_on_submit
-    ClaimFastTracking::DiagnosticCodesForMetrics::DC.intersection(diagnostic_codes).each do |diagnostic_code|
-      next unless disabilities.any? do |dis|
-        diagnostic_code == dis['diagnosticCode']
```

Comment:
The intersection filter and this `next unless` short-circuit (which are equivalent, actually) have been replaced by the `disability_claimed` boolean below -- instead of totally skipping over max-rated conditions that weren't submitted, we log the value of `disability_claimed` for extra visibility.